test/suite/test_tiered02.py

```diff
@@ -90,8 +90,11 @@ def test_tiered(self):
         ds.check()
         self.progress('checkpoint')
         self.session.checkpoint()
+
+        # We need to use sync=on because we check immediately
+        # to see that the files have arrived.
         self.progress('flush_tier')
-        self.session.flush_tier(None)
+        self.session.flush_tier('sync=on')
```

Comment:
This is very confusing to me. In `api_data.py` the default for `sync` is already `on`. This change should make no difference at all in the execution from the `None` case. Is there something unexpected going on with the config parsing? Can you investigate that?